crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -178,45 +172,45 @@ impl Reader {
     async fn initialize(
         &mut self,
     ) -> eyre::Result<((), executor::Handle<StateIsInit>, tendermint::chain::Id)> {
```

Comment:
I just noticed this function signature. This needs to be refactored in a followup. There is no reason to return an empty tuple out of all things.

Reply:
Made comment with followup issue linked.